tests/parm/nems.configure.cpld_xgrid.IN

```diff
@@ -0,0 +1,124 @@
+#############################################
+####  NEMS Run-Time Configuration File  #####
+#############################################
+
+# ESMF #
+logKindFlag:            ESMF_LOGKIND_MULTI
+
+# EARTH #
+EARTH_component_list: MED ATM OCN ICE
+EARTH_attributes::
+  Verbosity = 0
+::
+
+# MED #
+MED_model:                      @[med_model]
+MED_petlist_bounds:             @[med_petlist_bounds]
+::
+
+# ATM #
+ATM_model:                      @[atm_model]
+ATM_petlist_bounds:             @[atm_petlist_bounds]
+ATM_attributes::
+  Verbosity = 0
+  DumpFields = false
+  ProfileMemory = false
+  OverwriteSlice = true
+::
+
+# OCN #
+OCN_model:                      @[ocn_model]
+OCN_petlist_bounds:             @[ocn_petlist_bounds]
+OCN_attributes::
+  Verbosity = 0
+  DumpFields = false
+  ProfileMemory = false
+  OverwriteSlice = true
+  mesh_ocn = @[MESHOCN_ICE]
+::
+
+# ICE #
+ICE_model:                      @[ice_model]
+ICE_petlist_bounds:             @[ice_petlist_bounds]
+ICE_attributes::
+  Verbosity = 0
+  DumpFields = false
+  ProfileMemory = false
+  OverwriteSlice = true
+  mesh_ice = @[MESHOCN_ICE]
+  stop_n = @[RESTART_N]
+  stop_option = nhours
+  stop_ymd = -999
+::
+
+# CMEPS warm run sequence
+runSeq::
+@@[coupling_interval_slow_sec]
+   MED med_phases_prep_ocn_avg
+   MED -> OCN :remapMethod=redist
+   OCN
+   @@[coupling_interval_fast_sec]
+     MED med_phases_aofluxes_run
+     MED med_phases_prep_atm
+     MED med_phases_prep_ice
+     MED -> ATM :remapMethod=redist
+     MED -> ICE :remapMethod=redist
+     ATM
+     ICE
+     ATM -> MED :remapMethod=redist
+     MED med_phases_post_atm
+     ICE -> MED :remapMethod=redist
+     MED med_phases_post_ice
+     MED med_phases_prep_ocn_accum
+   @
+   OCN -> MED :remapMethod=redist
+   MED med_phases_post_ocn
+   MED med_phases_restart_write
+   MED med_phases_history_write
+@
+::
+
+# CMEPS variables
+
+DRIVER_attributes::
+::
+
+MED_attributes::
+      ATM_model = @[atm_model]
+      ICE_model = @[ice_model]
+      OCN_model = @[ocn_model]
+      history_n = 1
+      history_option = nhours
+      history_ymd = -999
+      coupling_mode = @[CPLMODE]
+      history_tile_atm = @[ATMTILESIZE]
+      aoflux_grid = 'xgrid'
+      aoflux_code = 'ccpp'
+      aoflux_ccpp_suite = 'FV3_sfc_ocean'
+      ccpp_restart_interval = 21600
```

Comment:
Why does this need to be 6 hours? The test is running for 24hrs, the components are writing restarts every 12 hours. Is the 6 hour restart being used, or is this just for testing? If the extra restart isn't needed, maybe the ccpp_restart_interval should be using RESTART_N?

Reply:
I could change it to 12 hours. It is just for testing since it also test CCPP specific I/O code in CMPES side. Of course, this could be  set by RESTART_N. Do you want me to change it?

Reply:
@DeniseWorthen it is done.